tests/fv3_conf/cpld_docn_cdeps_mom6.IN

```diff
@@ -0,0 +1,10 @@
+if [ -d "INPUT_DOCN" ]; then
+  echo "INPUT_DOCN/ directory exists ..."
+else
+  echo "INPUT_DOCN/ directory not found. Creating ..."
+  mkdir -p INPUT_DOCN 
+fi
+
+cd INPUT_DOCN
+rsync -arv @[RTPWD]/DOCN_MOM6_input_data/. .
```

Comment:
can we merge these little cpld_docn_cdeps into one file and use a flag to control the data sets?

Reply:
Again, currently renamed these hafs related fv3_conf files with a hafs_ prefix to reduce confusion. In the future, will further unify and clean up the HAFS related regression tests.